estuary/utils/general.py

```diff
@@ -399,3 +399,37 @@ def set_story_labels(requested_node_label, results, reverse=False):
         idx += delta
         node_idx += delta
     return results
+
+
+def create_header(item_type, item_id, story_node_story_flow, backward):
+    """
+    Generate a header for the siblings chart.
+
+    :param string item_type: the name of the type of current item
+    :param string item_id: the id of the current item
+    :param dict story_node_story_flow: has forward and backward relationships of current item
+    :param bool backward: determines if the results are in the reverse order of the story flow
+    :return: returns the appropriate siblings chart header
+    :rtype: string
+    """
+    def _plural_label(label):
+        if label.endswith('y'):
+            label = label[:-1] + 'ies'
+        else:
+            label = label + 's'
+        return label
+
+    if backward and story_node_story_flow['forward_label']:
+        fwd_label = _plural_label(story_node_story_flow['forward_label'])
+        fwd_relationship = \
+            story_node_story_flow['forward_relationship'][:-1].lower().replace('_', ' ')
+        result = '{0} {1} {2} {3}'.format(fwd_label, fwd_relationship, item_type, item_id)
```

Comment:
I don't think this will always be proper grammar based on the relationship names. You'll have to write a function that makes the relationship names consistent grammatically. Perhaps you could add these values in the return value of `story_flow` like `forward_relationship_display`.